rock/stream_aligner_test_helpers.rb

```diff
@@ -0,0 +1,89 @@
+# frozen-string-literal: true
+
+# no-doc
+module Syskit
+    module Aggregator
+        # Test helpers for stream aligner
+        #
+        # These create a synchronization by using the `transformer_stream_aligner_port`
+        # for transformer tasks or `stream_aligner_status_port` for pure stream aligner
+        # tasks by checking the time of the processed samples until one matches the
+        # written sample time or the timeout is reached.
+        module TestHelpers
+            # Confirms that the sample has been processed by the stream aligner using time
+            # comparisons between sample and processed samples.
+            # Other predicates can be used inside the expect execution.
+            #
+            # @param task
+            # @param port The port where the sample must be written
+            # @param sample The sample that must be written
+            # @param [String] sample_time_field name of the field in the sample that
+            # contains the logical time
+            # @param [&block] Extra predicates needed for the test
```

Comment:
```suggestion
            # @param [block] Extra predicates needed for the test
```

Saving it for now in case there are more changes to be made to avoid messing up the git history.